app/styles/site-views/search-list.scss

```diff
@@ -0,0 +1,80 @@
+$app-qgreen: rgba(98,144,128,0.25);
```

Comment:
please look below at my comments and review the nesting of child components in parent components. http://sass-lang.com/documentation/file.SASS_REFERENCE.html#nested_rules

Reply:
done